scripts/install/linux_compilation_dependencies.sh

```diff
@@ -11,6 +11,8 @@ apt install --yes git lsb-release cmake swig libglu1-mesa-dev libglib2.0-dev lib
 UBUNTU_VERSION=$(lsb_release -rs)
 if [[ $UBUNTU_VERSION == "20.04" ]]; then
        apt install --yes libzip5
+elif [[ $UBUNTU_VERSION == "22.04" ]]; then
+       apt install --yes libzip4
```

Comment:
Is it normal we install libzip4 on 22, and libzip5 on 20?

Reply:
Yes, this is the default library in Ubuntu 22.04.